server.js

```diff
@@ -19,7 +46,64 @@ app.use(express.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Hello world, welcome to Happy Thoughts API - made by Darya // See this API live in action here: https://happy-thoughts-react-project.netlify.app/')
+});
+
+// getting all of the thoughts and sorting them
+app.get('/thoughts', async (req, res) => {
+  const { page = 1, limit = 5 } = req.query;
+
+  try {
+    const thoughts = await Thought.find({})
+      .limit(limit * 1)
```

Comment:
nice limit! What dose the (limit * 1) do? and then together with the .skip((page - 1) * limit) :D